src/map/map.c

```diff
@@ -2840,63 +2848,70 @@ static int map_mapname2ipport(unsigned short name, uint32 *ip, uint16 *port)
 	return 0;
 }
 
-/*==========================================
+/**
  * Checks if both dirs point in the same direction.
- *------------------------------------------*/
-static int map_check_dir(int s_dir, int t_dir)
+ * @param s_dir: direction source is facing
+ * @param t_dir: direction target is facing
+ * @return 0: success(both face the same direction), 1: failure
+ **/
+static int map_check_dir(enum unit_dir s_dir, enum unit_dir t_dir)
```

Comment:
Use `uint8` instead of `enum unit_dir`, since `uint8` uses just 1 byte of memory, while `enum unit_dir` would use 4 bytes.
`static int map_check_dir(uint8 s_dir, uint8 t_dir)`

Reply:
No, see @hemagx ' reply